src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerTeamController.cs

```diff
@@ -482,14 +482,21 @@ private async Task<OrchestratorResponse<AccountDashboardViewModel>> GetAccountIn
             return response;
         }
 
-        private bool HasPayeScheme(AccountDashboardViewModel data)
+        private bool HasPayeScheme(AccountDashboardViewModel accountViewModel)
         {
-            return data.PayeSchemeCount > 0;
+            return accountViewModel.PayeSchemeCount > 0;
         }
 
-        private bool HasOrganisation(AccountDashboardViewModel data)
+        private bool HasOrganisation(AccountDashboardViewModel accountViewModel)
         {
-            return data.OrgainsationCount > 0;
+            return accountViewModel.OrganisationCount > 0;
+        }
+
+        private bool DisplayNewHomepage(AccountDashboardViewModel accountViewModel)
+        {
+            return FeatureToggles.Features.HomePage.Enabled
```

Comment:
Can you confirm that we should display the new homepage even when the the feature isn't enabled?

Reply:
yes the logic is, new homepage should be displayed at all times when toggled on i.e in DEMO.

When it is toggled off we want to direct the users to new homepage to prompt them to add their PAYE (when skipped) and to prompt them to sign their agreement (new)

Reply:
👍